TodoList/Models/Item.swift

```diff
@@ -2,10 +2,12 @@ import Foundation
 
 struct Item: Codable {
     let title: String
+    var completionDate: Date?
     var isComplete: Bool
```

Comment:
I would expect `isComplete` to be calculated property of `completionDate`.
This can introduce inconsistency between those two.

Reply:
That's a great point @stemberamichal. I'll change that when I get a chance. Thanks!!

Reply:
I created an issue [(GH-30)](https://github.com/chuva-io/TodoList/issues/30)  to deal with this. I don't want to introduce it in this PR that is focused on other topic.